docs/USAGE-GUIDE.md

```diff
@@ -1658,6 +1723,7 @@ Once the output is set the asynchronous private key operation can be completed b
 following the steps outlined [above](#Asynchronous-private-key-operations-related-calls)
 to apply the operation and free the op object.
 
+The [handshake algorithm getters](###S2N-Get-Selected-Handshake-Algorithms) and [certificate context](#s2n\_cert\_chain\_and\_key\_get\_ctx) APIs can be helpful for determining the private key and what private key algorithm to use.
```

Comment:
I prefer the phrasing, "The handshake algorithm getters and certificate context APIs can be used to determine the private key and which private key algorithm to use."